admin/listPurchases/listView.html

```diff
@@ -77,13 +88,142 @@
             tbody.innerHTML = "<tr><td colspan='1000'>Fejl ved load af ordrer. Prøver igen automatisk. Alternativt kan du forsøge at opdatere siden (tryk F5).</td></tr>";
         }
         else {
-            renderEntriesToList(entries.filter(option.filter));
+            currentPageIndex = 0;
+            currentOption = option;
+            updateContent();    
         }
-
         return false;
     });
 });
 
+const amountOfShownSubpages = 5;
+const entriesPerSubpage = 20;
+let amountOfSubpages;
```

Comment:
Generally global scope variables for sharing state are an antipattern - you should prefer passing the value to functions that need it.

So seeing this global variable here means we should think about the data flow: why is it used here, and could we restructure it?

The problem with global variables is that it makes code much harder to reason about, because lots of sources can modify or react to the variable, and it isn't clear from the function signatures (name + parameters) if they use it.

For the two above constants, it's okay that they are global: they cannot change, so the potential complexity isn't increased in the same way.